src/osdp_cp.c

```diff
@@ -318,7 +318,6 @@ static int cp_build_command(struct osdp_pd *pd, uint8_t *buf, int max_len)
 		if (smb == NULL) {
 			break;
 		}
-		osdp_get_rand(pd->sc.cp_random, 8);
```

Comment:
I wonder if this has any security implications.

But then, if the standard says so.. :)

Reply:
I wondered that too.  In the case of the busy reply I figured it would not be a problem because we are re-sending the exact same packet (with the same sequence number and checksum/CRC) and so nothing additional is being revealed on each send.  I'm not sure though that covers all cases.